{"name": "test_login_fail[data2]", "status": "passed", "attachments": [{"name": "log", "source": "ad91028c-5a37-4e14-b365-313792ecdf70-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'id': 3, 'title': '6位错误验证码', 'url': 'http://192.168.3.157:8069/dongwang-app/app/user/login', 'json': {'type': '1', 'flashToken': '', 'deviceId': 'aaa', 'device': '1', 'smsCode': '123456', 'phone': '113157121093'}, 'expected': {'status_code': 500, 'message': '验证码输入错误,请重新输入'}}"}], "start": 1614590581314, "stop": 1614590581346, "uuid": "8344e9e7-88ab-4cc2-bae4-9013cd493c6a", "historyId": "490f94efd3791f18d8a00ee815ee620a", "testCaseId": "a67639c7d108120b92ca3b258a406cc7", "fullName": "testcase.test_login.TestLogin#test_login_fail", "labels": [{"name": "story", "value": "错误验证码"}, {"name": "feature", "value": "登录模块"}, {"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "SC-201907251023"}, {"name": "thread", "value": "3528-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_login"}]}